great_expectations/analytics/client.py

```diff
@@ -56,8 +57,13 @@ def init(
         conf["oss_id"] = oss_id
     update_config(config=Config(cloud_mode=cloud_mode, **conf))
 
-    posthog.disabled = not ENV_CONFIG.posthog_enabled
-    if ENV_CONFIG.posthog_enabled:
+    # Analytics are enabled by default
+    # A user can opt out through the config or by setting the environment variable
+    # If one is set to False, analytics will be disabled
+    enabled_from_config = True if enabled_from_config is None else enabled_from_config
```

Comment:
Why aren't we just defining the param as `enabled_from_config: bool = True`? Maybe I'm missing context, but why does it being enabled from config determine anything about analytics?